modules/nw-endpoint-route53.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="nw-endpoint-route53_{context}"]
+= Endpoint configuration for route53
+
+The `route53` endpoint URL must use regions, `us-east-1`, `us-west-1`, or no region because Route 53 is not a regionalized service. The `us-east-1` region is required for the `tagging` endpoint URL to get the hosted zone, `route 53`, of the Elastic Load Balancing (ELB).
+
+[IMPORTANT]
+====
+Private, disconnected installs are not supported for AWS GovCloud when installing in the `us-gov-east-1` region.
```

Comment:
I'm not aware of this limitation. Users should be able to install in either GovCloud region without the need to specify any custom endpoints.

Reply:
Thank you for reviewing, @danehans! I pulled the note from the known issue that came up on Friday of last week: https://github.com/openshift/openshift-docs/pull/26300, but am happy to remove it. 

Do you think this content is important for the user when installing on GovCloud? The intention for this module is to include it in the AWS GovCloud installation guide. 

Currently the into paragraph is: 
The `route53` endpoint URL must use regions `us-west-1. The `us-west-1` region is required for the `tagging` endpoint URL to get the hosted zone, `route 53`, of the Elastic Load Balancing (ELB).

Would it be better to include your breakdown or replace the introduction all together: 
"If you install in either GovCloud region, Ingress Operator uses `us-gov-west-1` region for Route53 and tagging API clients.

Ingress Operator uses `https://tagging.us-gov-west-1.amazonaws.com` as the tagging API endpoint if a tagging custom endpoint is configured that includes the string 'us-gov-east-1'."

Reply:
> Thank you for reviewing, @danehans! I pulled the note from the known issue that came up on Friday of last week: #26300, but am happy to remove it.

@bmcelvee thanks for the heads-up. I was unaware of that bz, so please keep the note.

> Do you think this content is important for the user when installing on GovCloud? The intention for this module is to include it in the AWS GovCloud installation guide.

I don't think so, the caveats stated are specific to "custom endpoints" and according to @abhinavdahiya custom endpoints are not required for GovCloud.

> Would it be better to include your breakdown or replace the introduction all together:
"If you install in either GovCloud region, the Ingress Operator uses us-gov-west-1 region for Route53 and tagging API clients.

Yes, I like this replacement.